src/main/webapp/app/exercises/quiz/manage/match-percentage-info-modal/match-percentage-info-modal.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+    selector: 'jhi-match-percentage-info-modal',
+    templateUrl: './match-percentage-info-modal.component.html',
+})
+export class MatchPercentageInfoModalComponent implements OnInit {
+    constructor(private modalService: NgbModal) {}
+
+    /**
+     * Do nothing on init.
+     */
+    ngOnInit() {}
```

Comment:
Not sure here but if you don't need it you can remove that and remove the `ìmplements OnInit`